hw/xbox/xid-sdl.c

```diff
@@ -106,10 +105,9 @@ typedef struct USBXIDState {
 
     uint8_t device_index;
     SDL_GameController *sdl_gamepad;
```

Comment:
There is a lot of properties to this `struct` that are prefixed with `sdl`. Maybe I should add a new struct like `SDLDeviceState`(name not final) and have that as a property of `USBXIDState`?

Reply:
Just group them visually for now. We'll probably worry about this when doing #59